src/rebar3_gpb_plugin.app.src

```diff
@@ -11,7 +11,6 @@
         {env,[]},
         {modules, []},
         {pkg_name, nextroll_rebar3_gpb_plugin},
-        {maintainers, ["Luis Rascao"]},
```

Comment:
```suggestion
        {maintainers, ["NextRoll"]},
```

Reply:
`rebar3_hex` yells a bit at that, but sure
```
===> package argument given, will not publish docs
===> rebar3_gpb_plugin.app.src : deprecated field maintainers found
```